src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -141,106 +141,6 @@ public void BindingWorkflow_DownloadQualityProfile_Failure()
         }
 
         [TestMethod]
-        public void BindingWorkflow_SetSolutionRuleSet()
```

Comment:
The workflow is just responsible to call the solution operation API which is unit tested separetly